sssd_test_framework/utils/sssctl.py

```diff
@@ -171,3 +174,29 @@ def passkey_register(
             )
 
         return result.stdout_lines[-1].strip()
+
+    class Action(Enum):
+        """
+        Action parameter for ``user_checks``.
+        """
+        AUTH = "auth"
+        ACCT = "acct"
+        SETC = "setc"
+        CHAU = "chau"
+        OPEN = "open"
+        CLOS = "clos"
```

Comment:
I think it would be nicer to use  PAM function names instead of the shortcuts from `sssctl user-checks` argument parser, i.e. https://github.com/SSSD/sssd/blob/32f578229d38766b208f33130e28317ca69001d2/src/tools/sssctl/sssctl_user_checks.c#L268-L300

* pam_authenticate -> authenticate
* pam_setcred -> setcred
* ...

Reply:
But I wonder if enum is a good choice here, as it is useful only for user_checks but has a generic name "Action". This would create a conflict at some point with other command. We can rename it to UserChecksAction but that is long, so maybe we can remove the enumeration and just provide a string to the action argument.

What do you think?

Reply:
Thank you for your feedback. I do not have a strong opinion on this. I think the use of enum here comes in handy especially when the attributes are mostly abbreviations, that are easily typed incorrectly. However, I see your point about potential naming conflicts. So if we are keeping this enum, change of the name is necessary. So I would probably go with UserChecksAction, as you mentioned, although it is a bit long because I cannot think of better name. What do you think?

Reply:
I think `client.sssctl.user_checks(action="auth")` is better then `client.sssctl.user_checks(action=client.sssctl.UserChecksAction.AUTHENTICATE)`. Enum values would be better if we provide something for general use, but just passing the arguments to make the test case shorter and more intuitive (you want to execute sssctl user-checks command just in python tests so you follow the same arguments) is better choice.

Reply:
You already know the value of action parameter since you want to test specific command. You are not creating something new, just testing exact command.